examples/public.php

```diff
@@ -0,0 +1,61 @@
+<?php
+// Example - get all public wall links - useful for repost links pulbics
+session_start();
+
+require '../vendor/autoload.php';
+
+use \BW\Vkontakte as Vk;
+
+$vk = new Vk([
+    'client_id' => 'XXXXXX',
+    'client_secret' => 'XXXXXX',
+    'redirect_uri' => 'http://localhost/vkontakte-php-sdk/examples/',
+]);
+$public = '';
+
+if (isset($_GET['code'])) {
+    $vk->authenticate();
+    $_SESSION['access_token'] = $vk->getAccessToken();
+    header('Location: ' . $_SERVER['REQUEST_SCHEME'] . '://' . $_SERVER['HTTP_HOST'] . $_SERVER['PHP_SELF']);
+    exit;
+} else {
+    $vk->setAccessToken($_SESSION['access_token']);
+}
+
+$userId = $vk->getUserId();
+
+$wall0Empty = $vk->api('wall.get', [
+    'domain' => $public,
+]);
+$wallCount = $wall0Empty['count'];
+
+$apiParams = [
+    'domain' => 'xxxxxxxx',
+    'count' => 100
+];  
+
+$wallArray = [];
+for($i = 0; $i < ceil($wallCount / 100); $i++) {
+    if($i > 0) {
```

Comment:
I know, we don't use PHP CS Fixer in this project yet (but we should add it some day! - a good issue for a new PR btw), but let's stick a good coding standards and add a one space between `if` statement and parenthesis, it should be:
``` php
if ($i > 0) {
```

It should be fixed in a few other places in your code near some `if` / `for` / `foreach` statements. P.S. [PSR-1 / PSR-2](http://www.php-fig.org/) is good for addinional info about it